server/src/pages/shot/time-diff.js

```diff
@@ -24,6 +24,12 @@ exports.TimeDiff = class TimeDiff extends React.Component {
     }
   }
 
```

Comment:
The `LEN` function is not available in `fluent`, but selectors still work: http://projectfluent.io/fluent/guide/selectors.html